tools/api_check/test/apicheck_test.dart

```diff
@@ -39,26 +43,27 @@ checkApiConsistency(String flutterRoot) {
   test('AccessibilityFeatures enums match', () {
     // Dart values: _kFooBarIndex = 1 << N
     List<String> uiFields = getDartClassFields(
-      sourcePath:'$flutterRoot/lib/ui/window.dart',
-      className:'AccessibilityFeatures',
+      sourcePath: platformize('$flutterRoot/lib/ui/window.dart'),
```

Comment:
Should we convert these to `path.join`s instead? That seems more idiomatic

Reply:
Thanks, I made the change